eo-maven-plugin/src/main/java/org/eolang/maven/AssembleMojo.java

```diff
@@ -203,14 +203,15 @@ public void exec() throws IOException {
             new Moja<>(PlaceMojo.class),
         };
         while (true) {
+            final StopWatch times = new StopWatch();
```

Comment:
@mximp what's the point of this class? why can't you just do this:

```
long start = System.currentTimeMillis();
// ...
Logger.info("Tooks %[ms]s", System.currentTimeMillis() - start);
```

that's it

Reply:
@yegor256 The class does slightly more than just calculating the diff. It collects several time points (laps) since start which later can be retrieved. I used this during tuning performance.
Also I believe this object will be helpful further for performance optimisations in eo-runtime (we might want to extend it).
Besides isn't it better to encapsulate this logic in an object? Client code doesn't need to know how we get instants and calculate duration.